server/src/main/java/org/elasticsearch/rest/action/search/RestMultiSearchAction.java

```diff
@@ -72,6 +80,9 @@ public String getName() {
 
     @Override
     public RestChannelConsumer prepareRequest(final RestRequest request, final NodeClient client) throws IOException {
+        // set weather data nodes should send back stack trace based on the `error_trace` query parameter
+        context.putHeader("error_trace", request.param("error_trace", "false"));
```

Comment:
do we have a constant for error_trace that we could point to?

Reply:
I added a constant and modified the already existing paramAsBoolean function calls for `error_trace, that takes the default, to use the new constant. (I could also make this a separate PR, but the change is quite small)